templates/user-research/index.html

```diff
@@ -0,0 +1,201 @@
+{% extends "telco/base_telco.html" %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1gOoOJNnrUVWOhpYlkShq0FAbMqDZsj-CXuOGT_S8Xl4/edit#heading=h.hfhqccyix5vw{% endblock meta_copydoc %}
+
+{% block title %}Canonical User Research Programme{% endblock %}
+
+{% block meta_description %}Ubuntu is a leading platform in the telecommunications sector. Ubuntu powers telecommunications infrastructure of leading global service providers. Ubuntu is in the heart of the entire ETSI NFV stack, providing both NFVI and MANO, bringing operational costs down and ensuring the highest level of security. All of that at a competitive price.{% endblock %}
+
+{% block content %}
+
+<div class="is-paper">
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h1 class="p-muted-heading">Canonical <br>user research</h1>
+      </div>
+      <div class="col-9 col-medium-4">
+        <p class="p-heading--2 p-block"><strong>Help make Canonical products better.</strong><br>Share your thoughts and experiences.</p>
+        <p class="p-block">You'll get  early access to the products Canonical's engineers and designers are working on right now &ndash; we love to collaborate with users when creating new features and designs. Your insights will influence our direction, and everyone's user experience.</p>
+        <p class="p-block"><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp" class="p-button--positive">Join our User Research Program</a></p>
+          {{ image (
```

Comment:
Not sure what the convention is with the new design, should this be hidden on small screens?

Reply:
My thinking with stacked images like this is that they don't need to be hidden, and it wasn't flagged in design review so I left it as is. Wouldn't harm to get @lyubomir-popov 's input though?